Library/Formula/voikko-fi.rb

```diff
@@ -0,0 +1,15 @@
+class VoikkoFi < Formula
+  desc "Finnish dictionary for Voikko speller"
+  homepage "http://voikko.puimula.org"
+  url "http://www.puimula.org/voikko-sources/voikko-fi/voikko-fi-2.0.tar.gz"
+  sha256 "02f7595dd7e3cee188184417d6a7365f9dc653b020913f5ad75d1f14b548fafd"
+
+  depends_on "python3" => :build
+  depends_on "foma" => :build
+  depends_on "libvoikko"
+
+  def install
+    system "make", "vvfst"
+    system "make", "vvfst-install", "DESTDIR=#{HOMEBREW_PREFIX}/lib/voikko"
```

Comment:
Is it OK to link under libvoikko directory from voikko-fi, or do we need a separate shared directory for dictionaries? Both are possible, voikko-fi does installation under libvoikko by default.